packages/polkadart_scale_codec/test/functions_test/extensions_test.dart

```diff
@@ -3,40 +3,53 @@ import 'package:polkadart_scale_codec/polkadart_scale_codec.dart';
 
 void main() {
   group('IntConverter Extension:', () {
-    test(
-        '5 as int will be converted to BigInt.from(5) when used with IntConverter Extension',
+    test('When provided with 5 as int, it will result in an BigInt Object.',
         () {
       final bigInt = 5.toBigInt;
       expect(BigInt.from(5), equals(bigInt));
     });
-    test(
-        '500 as int will be converted to BigInt.from(500) when used with IntConverter Extension',
+
+    test('When provided with 0 as int, it will result in an BigInt Object.',
+        () {
+      final bigInt = 0.toBigInt;
+      expect(BigInt.from(0), equals(bigInt));
+    });
+
+    test('When provided with 500 as int, it will result in an BigInt Object.',
```

Comment:
Isn't this testing the same as `When provided with 5 as int, it will result in an BigInt Object.`?
Also, our descriptions should be about the code's behavior, not it's structure. `BigInt` is a tool we're using for handling with large objects: it's related to structure. `500` is a positive integer we're using to validate a test case, it's releated to the structure .

**This comment applies to all test messages** (I can point each of them out if that helps)
```suggestion
    test('When provided with a positive integer, it should convert to a big integer',
```